pkg/antctl/raw/multicluster/create/access_token.go

```diff
@@ -0,0 +1,198 @@
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"antrea.io/antrea/pkg/antctl/raw"
+	multiclusterscheme "antrea.io/antrea/pkg/antctl/raw/multicluster/scheme"
+)
+
+type accessTokenOptions struct {
+	namespace      string
+	serviceAccount string
+	roleBinding    string
+}
+
+var accessTokenOpts *accessTokenOptions
+
+var accessTokenExamples = strings.Trim(`
+# Create an access token in a leader cluster for one or more member clusters. If the ServiceAccount or RoleBinding does not exist, the command will create them too
+  $ antctl mc create accesstoken <NAME> -n <NAMESPACE> --service-account <SERVICE_ACCOUNT> --role-binding <ROLE_BINDING>
+`, "\n")
+
+func (o *accessTokenOptions) validateAndComplete() error {
+	if o.namespace == "" {
+		return fmt.Errorf("the Namespace cannot be empty")
+	}
+	if o.serviceAccount == "" {
+		return fmt.Errorf("the ServiceAccount cannot be empty")
+	}
+	if o.roleBinding == "" {
+		return fmt.Errorf("the RoleBinding cannot be empty")
+	}
+	return nil
+}
+
+func NewAccessTokenCmd() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "accesstoken",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Create an accesstoken in a leader cluster",
+		Long:    "Create an accesstoken in a leader cluster",
+		Example: accessTokenExamples,
+		RunE:    accessTokenRunE,
+	}
+
+	o := &accessTokenOptions{}
+	accessTokenOpts = o
+	command.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Namespace of the ClusterClaim")
+	command.Flags().StringVarP(&o.serviceAccount, "service-account", "", "", "ServiceAccount of the access token")
+	command.Flags().StringVarP(&o.roleBinding, "role-binding", "", "", "RoleBinding of the ServiceAccount")
+
+	return command
+}
+
+func accessTokenRunE(cmd *cobra.Command, args []string) error {
+	if err := accessTokenOpts.validateAndComplete(); err != nil {
+		return err
+	}
+	if len(args) != 1 {
+		return fmt.Errorf("exactly one NAME is required, got %d", len(args))
+	}
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+	restconfigTmpl := rest.CopyConfig(kubeconfig)
+	raw.SetupKubeconfig(restconfigTmpl)
+	k8sClient, err := client.New(kubeconfig, client.Options{Scheme: multiclusterscheme.Scheme})
+	if err != nil {
+		return err
+	}
+
+	serviceAccount := corev1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      accessTokenOpts.serviceAccount,
+			Namespace: accessTokenOpts.namespace,
+		},
+	}
+
+	var createErr error
+	fmt.Fprintf(cmd.OutOrStdout(), "Creating ServiceAccount \"%s\"\n", accessTokenOpts.serviceAccount)
+	createErr = k8sClient.Create(context.TODO(), &serviceAccount)
+	if createErr != nil {
+		if errors.IsAlreadyExists(createErr) {
+			fmt.Fprintf(cmd.OutOrStderr(), "The ServiceAccount \"%s\" already exists\n", accessTokenOpts.serviceAccount)
+			createErr = nil
+		} else {
+			return createErr
```

Comment:
Will the caller log the returned error? Maybe better to log here.

Reply:
The error will be thrown out by the antctl in the terminal. Do you mean that we should use a log method such `klog` to log the error here?

Reply:
I meant should we use Printf() to show that ServiceAccount creation failed for what error. I assume the message printed out by the caller, does not have information what exact step failed?

Reply:
Added more information to show ServiceAccount failed error, as well as RoleBingding.